src/components/ExpenseItem/ExpenseItem.jsx

```diff
@@ -10,30 +10,15 @@ import {
 
 import { NoteIcon } from '@/assets';
 
-export const ExpenseItem = ({ transaction, onEdit, onDelete }) => {
+export const ExpenseItem = ({ transaction, onEdit, onDelete, version }) => {
```

Comment:
do it boolean? like `isShortView` or `isLongDisplayVersion` or something like that

Reply:
done